File: project/bootstrap/src/main/java/com/numberbox/BootstrapApplication.java
package com.numberbox;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "com.hexagonal")
public class BootstrapApplication {
    public static void main(String[] args) {
        SpringApplication.run(BootstrapApplication.class, args);
    }
}

File: project/bootstrap/src/main/kotlin/com/kamcci/numberbox/BootstrapApplication.kt
package com.kamcci.numberbox

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ComponentScan
import org.springframework.scheduling.annotation.EnableScheduling

@ComponentScan(basePackages = ["com.kamcci.modules", "com.kamcci.numberbox"])
@EnableScheduling
@SpringBootApplication
class BootstrapApplication

fun main(args: Array<String>) {
    runApplication<BootstrapApplication>(*args)
}
